app/src/androidTest/java/ch/epfl/sdp/blindly/profile_setup/ProfileBirthdayTest.kt

```diff
@@ -25,7 +25,7 @@ import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 
-const val TEST_BIRTHDAY = "18.03.2003"
+const val TEST_BIRTHDAY = "16.05.2003"
```

Comment:
What's the point of changing the date ? Both are > 18 no ?

Reply:
Yes, indeed, but it was to test the corner case of m == month and d <= day in getUserAge()